js/keyboard/keyboard.js

```diff
@@ -0,0 +1,147 @@
+import {
+	LANGUAGE,
+} from '../constants.js';
+import buttons from './buttonData.js';
+import Button from './button.js';
+
+export default class Keyboard {
+	constructor(outputEl, language) {
+		this.language = language || LANGUAGE.EN;
+		this.outputEl = outputEl;
+		this.buttons = [];
+		this.isShifted = false;
+		this.el = document.createElement('div');
+	}
+
+	draw() {
+		this.el.classList.add('keyboard');
+		buttons.forEach((row) => {
+			const rowEl = document.createElement('div');
+			this.el.append(rowEl);
+			rowEl.classList.add('row');
+			row.forEach((buttonOpts) => {
+				const button = new Button({
+					language: this.language,
+					...buttonOpts,
+				});
+				this.buttons.push(button);
+				rowEl.append(button.draw());
+			});
+		});
+
+		return this.el;
+	}
+
+	attachEvents() {
```

Comment:
The method `attachEvents` contains functions which are not related to the events directly. It would be better to collect the events in this method  but callbacks' realization and called functions (`addCharacter`, `removeCharacter`, `toggleKeys`) move outside